src/actions/component-remove.js

```diff
@@ -256,8 +257,11 @@ export function removeSequence(activesComponents, deletedComponent) {
  */
 export function remove(activesComponents, idDeletedComponent) {
   const deletedComponent = activesComponents[idDeletedComponent];
-
-  if (deletedComponent.children.length === 0) {
+  
+  if(isLoop(deletedComponent)) {
+    return removeComponentFromActivesComponent(activesComponents, deletedComponent);
+  }
+  if (!isLoop(deletedComponent) && deletedComponent.children.length === 0) {
```

Comment:
I think `!isLoop(deletedComponent)` is useless because you have the opposite if right before with a return statement,